### ISSUE TITLE
Does not work on mac M1

### ISSUE THREAD

**Issue by Syazvinski** [lacymorrow/crossover#93]
<img width="242" alt="Screen Shot 2021-07-20 at 1 49 23 AM" src="https://user-images.githubusercontent.com/28970765/126237288-a167ec64-68d2-48c4-8363-af2a36f5a080.png">

Ive tried both files and both give me errors.

**Comment by lacymorrow** [comment]
Dang. Thanks for testing, I'm assuming you tried both 'arm64' versions? 
I appreciate you testing, I'll probably get an M1 mac for work at some point and will be able to debug but for now, if it's not being built correctly I can't fix it.

https://github.com/lacymorrow/crossover/releases/download/latest/CrossOver-2.1.1-arm64.zip
https://github.com/lacymorrow/crossover/releases/download/latest/CrossOver-2.1.1-arm64.dmg

**Comment by Syazvinski** [comment]
Yes, I have tried it on both versions. Im not sure what the problem is.  Let me know if you have new builds you want me to test.

**Comment by lacymorrow** [comment]
https://github.com/electron-userland/electron-builder/issues/5392#issuecomment-725779853

It looks like the latest preview of electron-builder supports building for M1 mac, although the builtin native node modules likely wont work (iohook, mouse hooks).

I'll need to update electron and work from there.

**Comment by lacymorrow** [comment]
Could you test this version out @Syazvinski ?

https://github.com/lacymorrow/crossover/releases/tag/2.1.2-zeta

**Comment by Syazvinski** [comment]
I can confirm this works on the macbook air m1. The one made for intel mac was buggy and didnt change the crosshair alot of the time. 

I tried the .dmg and it works perfectly. Everything updates on time and its not buggy or laggy.

Congrats!
